manifests/eureka-example/core-modules.yaml

```diff
@@ -0,0 +1,108 @@
+name: eks
+path: git::https://github.com/awslabs/idf-modules.git//modules/compute/eks
+dataFiles:
+  - filePath: git::https://github.com/awslabs/idf-modules.git//data/eks_dockerimage-replication/versions/1.29.yaml?ref=release/1.6.0
+  - filePath: git::https://github.com/awslabs/idf-modules.git//data/eks_dockerimage-replication/versions/default.yaml?ref=release/1.6.0
+parameters:
+  - name: vpc-id
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: VpcId
+  - name: controlplane-subnet-ids
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: PrivateSubnetIds
+  - name: dataplane-subnet-ids
+    valueFrom:
+      moduleMetadata:
+        group: optionals
+        name: networking
+        key: PrivateSubnetIds
+  - name: eks-admin-role-name
+    value: Admin
+  - name: eks-poweruser-role-name
+    value: PowerUser
+  - name: eks-read-only-role-name
+    value: ReadOnly
+  - name: eks-version
+    value: "1.29"
+  - name: eks-compute
+    value:
+      eks_nodegroup_config:
+        - eks_ng_name: ng-gpu
+          eks_node_quantity: 2
+          eks_node_max_quantity: 4
+          eks_node_min_quantity: 2
+          eks_node_disk_size: 100
+          eks_node_instance_type: "g5.2xlarge"
+          use_gpu_ami: True
+          eks_node_labels:
+            usage: gpu
+      eks_node_spot: False
+      eks_secrets_envelope_encryption: False
+      eks_api_endpoint_private: False
+  - name: eks-addons
```

Comment:
Can you specific about which drives you need and set the rest to` False`?

Reply:
Updated

Reply:
Yes, I only set the rest to  false